### ISSUE TITLE
Tooltips cannot be activated using the keyboard and not perceivable by screen readers

### ISSUE THREAD

**Issue by anirudhabhowmikhpe** [grommet/grommet#7874]
Tooltip should be opened when focus will be on tooltip by keyboard. It should announce the content of it by screen reader as well.

**More Description:**
Tooltips can be accessed via the keyboard, but can only be activated with the mouse, as no interactive
elements are used. Keyboard users are restricted in their use of the website and do not have access to
important information. The triggering element receives focus but cannot be activated. Screen readers can
reach and activate the element but cannot perceive the text that is displayed

**Example:** Tooltips cannot be displayed using the keyboard during account creation.

**Solution:** The information from tooltips should also be accessible via keyboard. Alternatively, important information
should not be hidden on the website but should be freely accessible.

**Please take reference these two tickets:**
https://hpe.atlassian.net/browse/GLCP-297225
https://hpe.atlassian.net/browse/GLCP-297236

![Image](https://github.com/user-attachments/assets/8805a129-f1e5-476b-b7e5-e5635972d102)
![Image](https://github.com/user-attachments/assets/8784dea6-92c7-4d6b-95d1-ee6c0c180a74)

**Comment by halocline** [comment]
Hi @anirudhabhowmikhpe - Thank you for this issue.

This is an implementation issue and cannot be resolved by the Grommet library.

Information critical to completing a task [should not be displayed in a tooltip](https://design-system.hpe.design/components/tip#accessibility).

Options:
1. **Strongly preferred** Use FormField's info messages. This is implemented for the password field in the [first example on the Forms page](https://design-system.hpe.design/components/tip#accessibility).
2. Use the [Popover pattern](https://design-system.hpe.design/templates/popover).

**Comment by jcfilben** [comment]
Marking as closed, see above comment
